src/main/java/uk/gov/justice/digital/delius/service/OffenderService.java

```diff
@@ -208,22 +206,5 @@ private Custody findCustodyOrThrow(Event activeCustodialEvent) {
                         .build());
     }
 
-    @Transactional
```

Comment:
Nice to see this move to its own service 👍🏿

Reply:
It's going to grow like a triffid in the next ticket unfortunately